CHANGELOG.md

```diff
@@ -1,4 +1,28 @@
 
+<!-- 8.0.5 START -->
+
+# 8.0.5 (2021-07-08)
+
+### Features and enhancements
```

Comment:
```suggestion
## Features and enhancements
```

Reply:
We have level 3 headings for these throughout the changelog. Is this something that we want to change?

Reply:
Yes. It should go from h1 to h2, not h1 to h3.

Reply:
Makes sense! We're going to merge this PR and finish up the release first though. Then we'll also have a look at updating Grot so it generates the correct headings, and fix the current ones.